truss/base/truss_spec.py

```diff
@@ -98,11 +98,11 @@ def memory(self) -> str:
 
     @property
     def memory_in_bytes(self) -> int:
-        return validate_memory_spec(self.memory)
+        return self.config.resources.memory_in_bytes
 
     @property
-    def use_gpu(self) -> str:
-        return self._config.resources.use_gpu
+    def use_gpu(self) -> bool:
+        return self._config.resources.use_gpu  # type: ignore[return-value]  # computed field.
```

Comment:
Doesn't the `computed_field` basically make it a property? Curious why these extra ignores are needed